### ISSUE TITLE
Collect 'evidence of implementation' for Usage Report

### ISSUE THREAD

**Issue by dr-shorthair** [w3c/sdw-sosa-ssn#272]
Use if possible
- OneWater #172 
- Australian geological surveys #171 
- OGC Connected Systems #170 
- OGC 3D Cadastre #169 
- OGC Iliad #168 
- KnowWhereGraph #167 
- SAREF #166 
- Australian Biodiversity Data Repository #161 
- TERN #160

**Comment by ldesousa** [comment]
Can an alignment be evidence of implementation? 
On which circumstance?
 - From SOSA/SSN to an external ontology/spec?
 - From an external ontology/spec to SOSA/SSN?

**Comment by ldesousa** [comment]
@dr-shorthair Would you welcome a PR adding [GloSIS](http://w3id.org/glosis/model) to the [list of ontologies](https://github.com/w3c/sdw-sosa-ssn/blob/f9f4d1bd31025c26481f5791cb760469cee56531/ssn/usage/index.html#L967)? Adding the respective statistics.

**Comment by dr-shorthair** [comment]
Yes, definitely

**Comment by dr-shorthair** [comment]
Is GloSIS still an option here?

**Comment by ldesousa** [comment]
> Is GloSIS still an option here?

Yes, my bad for not having contributed so yet. Is there a timeline? I can't pick it up before Easter.

**Comment by dr-shorthair** [comment]
> > Is GloSIS still an option here?
> 
> Yes, my bad for not having contributed so yet. Is there a timeline? I can't pick it up before Easter.

Its after Easter now! ;-) 

> Can an alignment be evidence of implementation?
> On which circumstance?

If the alignments are published externally, then we can use them as usage evidence (at least we did last time).
